### DISCUSSION TITLE
Tuya One Gang, Two Gang & Three Gang Switch

### DISCUSSION THREAD

**Discussion by smhmaqsood** [arendst/Tasmota#11997] [Q&A] [answered]
I'm having difficulty in configuring tasmota for two gang switch and three gang switch for Tuya (TWYE3S) MCU.
I've configured it for one gang but still two more to go. Can someone guide me about it.

**Comment by gck303** [comment, marked as answer]
I think you need to give a little more information. 

What version are you using?
How have you got it configured?
What is happening?
What would you expect to happen?
